lib/cimpler.js

```diff
@@ -95,9 +100,12 @@ function Cimpler(config) {
    this.registerMiddleware = function(route, inMiddleware){
       if (!connect) {
          connect = Connect()
-            .use(Connect.responseTime())
-            .use(Connect.bodyParser())
-            .use(Connect.query());
+            .use(responseTime())
+            .use(bodyParser({extended: true}))
```

Comment:
I'm not seeming to find this way of calling `bodyParser` in [the docs](https://www.npmjs.com/package/body-parser). Should this be
```suggestion
            .use(bodyParser.json({extended: true}))
```
or something like that?

Reply:
> body-parser deprecated bodyParser: use individual json/urlencoded middlewares

That ^ notification comes up in tests, but it seemed most similar to what we were doing before, and I was trying to limit the changes here as much as possible.

Reply:
Fair, although the fact that this technique is undocumented makes me concerned that it'll just break suddenly on some future upgrade.
Eh, I don't care that much.